docker/ambulance-analytics-sandbox/src/app.js

```diff
@@ -14,7 +14,18 @@ const init = async () => {
         relativeTo: Path.join(__dirname, 'mocks'),
       },
     },
-  })
+  });
+
+  server.route({
+    method: '*',
+    path: '/{any*}',
+    handler: function (request, h) {
+        const errorResponse = {
+          error: 'Server Not Found'
```

Comment:
Surely if this is a 404 the message should be "File not found", not "Server Not Found"?

Reply:
It's set to "Server Not Found" to mimic the real server; but I see how that's confusing.